src/components/ticker-bar/ticker-bar.ts

```diff
@@ -165,11 +169,12 @@ export class TickerBarComponent implements OnChanges {
     if (deltaMin > 59) {
       deltaStr =
         deltaHour > 1
-          ? String(deltaHour) + hour_str_multiple
-          : String(deltaHour) + hour_str_single
+          ? String(deltaHour) + ' ' + hour_str_multiple
+          : String(deltaHour) + ' ' + hour_str_single
```

Comment:
Would be good to use a library for date time manipulations like `moment.js` across the app. Something to add in a future refactor